src/parts/HeroImage/HeroImage.js

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import './HeroImage.css';
+import PropTypes from 'prop-types'
+
+const HeroImage = ({ className = '', children, image = '', alpha= 0.85 }) => {
+  const backgraundImage = `url("${image}")`;
+  const backgroundStyle = `linear-gradient(rgba(0, 0, 0, ${alpha}), rgba(0, 0, 0, ${alpha - .05})), ${backgraundImage}`;
+  return ( 
+    <div className={`HeroImage ${className}`} style={{backgroundImage: backgroundStyle}}>
+      <div className="HeroContent">
+        {children}
+      </div>
+    </div>
+  );
+};
+
+HeroImage.propTypes = {
+  className: PropTypes.string,
+  image: PropTypes.string,
+  alphaTop:PropTypes.number
```

Comment:
ez mostmár csak simán alpha , updateld amjd légyszi 👍

Reply:
Fixed in [41796b8](https://github.com/rkasza/netflix-roulette/pull/5/commits/41796b86e025ece8171a370f32275b9adfbd0b34)